frappe/public/js/frappe/file_uploader/FileUploader.vue

```diff
@@ -332,6 +332,89 @@ export default {
 		upload_file(file, i) {
 			this.currently_uploading = i;
 
+			if (frappe.get_cookie('aws_cognito_identity_id')) {
+				// this part should be done using async await,
+				// but I couldn't figure out how to use it properly
+				// ***
+				// tried:
+				// async upload_file(file, i) {
+				// 	console.log('test async #1')
+				// 	await frappe.db.get_doc('AWS S3 Attachment', 'AWS S3 Attachment');
+				// }
+				// or:
+				// upload_file: async (file, i) => {
+				// 	console.log('test async #2')
+				// 	await frappe.db.get_doc('AWS S3 Attachment', 'AWS S3 Attachment');
+				// }
+				// ***
+				// frappe won't load new scipt if i use async,
```

Comment:
Is there an error, or does it just not load at all?

Reply:
It just use the old script before i change to async function

Reply:
Ps. I did "Empty Cache and Hard Reload", it does detect change such as console.log but when I change it to async function it just doesn't work

Reply:
I tried pulling this locally to test, but I'm getting an error while setting it up.

Am I missing a step?

```js
Error: ENOENT: no such file or directory, open '/home/rohan/erpn/apps/frappe/frappe/public/js/lib/aws-sdk.min.js'
    at Object.openSync (fs.js:462:3)
    at Object.readFileSync (fs.js:364:35)
    at /home/rohan/erpn/apps/frappe/rollup/build.js:129:39
    at Array.map (<anonymous>)
    at /home/rohan/erpn/apps/frappe/rollup/build.js:123:36
    at Array.forEach (<anonymous>)
    at concatenate_files (/home/rohan/erpn/apps/frappe/rollup/build.js:120:18)
    at Object.<anonymous> (/home/rohan/erpn/apps/frappe/rollup/build.js:40:13)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/rohan/erpn/apps/frappe/frappe/public/js/lib/aws-sdk.min.js'
}

```

Reply:
I forgot to push aws-sdk.

I've push it now, please update your apps.

Reply:
Seems like adding async-await there causes a SyntaxError while building the file, and uses the original code instead. Not sure what could be causing the error though.